### DISCUSSION TITLE
InPaint API

### DISCUSSION THREAD

**Discussion by psydox** [lllyasviel/Fooocus#1519] [General]
Dear All,

I was testing the Gradio Client API for Fooocus.  I managed to generate an image using fn_index 30.  Now I am testing the InPaint.  How to send both the original image and mask via the fn_index 30?  There seem no arguments for the mask.

Thank you in advanced.

Kind regards

**Comment by IPv6** [comment]
https://github.com/konieshadow/Fooocus-API

**Reply by psydox** [reply]
Thanks for the link @IPv6 I appreciate it very much ^_^

Is there a direct way to do it via the fn_index ?
